cogs/TimeConv.py

```diff
@@ -0,0 +1,29 @@
+import discord
+from discord.ext import commands
+
+
+class TimeConverter(commands.Cog):
```

Comment:
For the sake of consistency, could you change the name of this class to `TimeCog` and change the file name to correspond to it? Thanks!